File: python3_solution_set2/2419. Longest Subarray With Maximum Bitwise AND.py
class Solution:
    def longestSubarray(self, nums: List[int]) -> int:
        
        ans = max(nums)
        c = 0
        result = 0
        for x in nums:
            if x == ans:
                c+=1
                
            else:
                c = 0
                
            
            result = max(result, c)
            
        return result

File: cpp_solutions/2419. Longest Subarray With Maximum Bitwise AND.cpp
class Solution {
public:
    int longestSubarray(vector<int>& nums) {
        int ans = *max_element(nums.begin(), nums.end());
        int c = 0;
        int result = 0;
        
        for(auto it : nums) {
            if (it == ans) {++c;}
            else{c=0;}
            result = max(result, c);
        }
        return result;
        
    }
};
